packages/core/src/common/uri-command-handler.ts

```diff
@@ -47,74 +46,65 @@ export interface MultiUriCommandHandler extends UriCommandHandler<URI[]> {
 
 }
 
-export namespace UriAwareCommandHandler {
-
+export class UriAwareCommandHandler<T extends MaybeArray<URI>> implements UriCommandHandler<T> {
     /**
-     * Further options for the URI aware command handler instantiation.
+     * @deprecated since 1.9.0. Please use `MonoSelectUriCommandHandler` or `MultiSelectUriCommandHandler`.
```

Comment:
Is `1.6.0` the default version for deprecation, even if it wasn't actually deprecated at that time?

Reply:
My reasoning is that the current version is 1.6.0 and we are deprecating starting now. It will eventually get removed/fixed in later versions. I think so far we've been waiting 2 minors, but maybe we should wait for the next major, I am not sure.

Also, perhaps I should have more explicitly stated that I wanted a change with the version but also the alternatives listed: `MonoSelectUriCommandHandler` alone doesn't exist anymore. Same for `MultiSelectUriCommandHandler`.

Reply:
Sorry, yes. 🤦